crates/uniswapx-rs/src/order.rs

```diff
@@ -90,33 +151,34 @@ impl ExclusiveDutchOrder {
         };
 
         // resolve over the decay curve
+        // TODO: apply cosigner logic
 
-        let input = ResolvedInput {
-            token: self.input.token.to_string(),
+        let input: ResolvedInput = ResolvedInput {
```

Comment:
dont think you need the type specification here since it's statically determinable